PGM/src/main/java/tc/oc/pgm/shop/currency/ExperienceCurrency.java

```diff
@@ -0,0 +1,48 @@
+package tc.oc.pgm.shop.currency;
+
+import net.md_5.bungee.api.chat.BaseComponent;
+import net.md_5.bungee.api.chat.TextComponent;
+import tc.oc.pgm.features.FeatureDefinition;
+import tc.oc.pgm.match.MatchPlayer;
+import tc.oc.pgm.shop.purchasable.Purchasable;
+
+/**
+ * A currency based on a {@link org.bukkit.entity.Player}'s XP level.
+ */
+public class ExperienceCurrency extends FeatureDefinition.Impl implements Currency {
```

Comment:
FeatureDefinitions are map related objects and should not interact with the match.